### PR TITLE
The view mode part should provide an api#5

### PR DESCRIPTION
**Pull request by KassandraA**
Pull request for view-model API task.

### CODE REVIEW

File: source/view-model.js [hunk 1]
```diff
@@ -4,14 +4,69 @@
...
+    };
+};
+
+let gamePreset = {
+    players: [
+        {type : 'human', name : 'First', symbol : 'X'},
+        {type : 'computer', name : 'Computer', symbol : 'O'}
+    ],
+
+    setPlayer: function(key, value){
```
Comment (aannenko) [inline source/view-model.js]: **Error**: looks like this method will find a specified key in the first player.
**Suggestion**: fix the error (make the method change player's properties based on the player's name) and rename the method to make it reflect its purpose better.
**Expected result**: `setPlayerProperty(playerName, propertyName, propertyValue)`, where `playerName` allows the method to find a player by their name, `propertyName` specifies which property to change and `propertyValue` specifies a value for this property. In the future tasks this method will be updated to find players by their IDs.

File: source/view-model.js [hunk 2]
```diff
@@ -4,14 +4,69 @@
...
+                if(i == key && item.hasOwnProperty(i)){
+                    item[i] = value;
+                    break;
+                }
+            }
+        });
+        return this.players;
+    },
+
+    getPlayer: function(){
```
Comment (aannenko) [inline source/view-model.js]: Please rename this method to `getPlayers` and add two more methods with the same name `getPlayer` that would return specific player by their name or number in the `players` array.

File: source/view-model.js (hunk 1 above)
Comment (KassandraA) [reply source/view-model.js]: Done

File: source/view-model.js (hunk 2 above)
Comment (KassandraA) [reply source/view-model.js]: Done. But you'd better see it first without pull request.

File: source/view-model.js [hunk 3]
```diff
@@ -4,14 +4,63 @@
...
+    this.getCurrentStatus = function(){
+        let gameStatus = game.currentStatus;
+        let playerName = gameStatus == 'Tie' ? 'It' : game.getCurrentPlayerName();
+        return  playerName + ' is ' + gameStatus;
+    };
+};
+
+let gamePreset = {
+    players: [
+        {type : 'human', name : 'First', symbol : 'X'},
```
Comment (aannenko) [inline source/view-model.js]: Could you please give the first player a better name ("Player", maybe)?

**Review by aannenko — CHANGES_REQUESTED** [review]

File: source/view-model.js (hunk 3 above)
Comment (KassandraA) [reply source/view-model.js]: Done

**Review by aannenko — APPROVED** [review]

**Pull request merged**